lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+extern crate js_sys;
+extern crate wasm_bindgen;
+extern crate vector2d;
+
+mod internal;
+
+use internal::{
+    emscripten_call_function,
+    msdfgen_generate_msdf,
+    msdfgen_free_font,
+    emscripten_data_types
+};
+use js_sys::Uint8Array;
+use wasm_bindgen::JsValue;
+use vector2d::Vector2D;
+use crate::internal::copy_f32_data_from_msdfgen_memory;
+
+// ============
+// === Font ===
+// ============
+
+pub struct Font {
+    handle: JsValue
+}
+
+impl Font {
+    // Loading font from memory
+    //
+    // Loads font from a any format which freetype library can handle.
+    // See https://www.freetype.org/freetype2/docs/index.html for reference
```

Comment:
This is not a doc comment. Doc comments start with `///`. Moreover, a minor thing - some time ago I was not thinking about it this way, but I learned from @iamrecursion to always put pride in creating nice docs, and it really pays off. So let's always remember about finishing sentences with dots etc. I know this may sounds too restrictive, but it pays off, as we want to keep the codebase really high quality.

Reply:
fixed